src/arplot.py

```diff
@@ -210,10 +210,12 @@ def plot_orderfits(slf, model, ydata, xdata=None, xmodl=None, textplt="Order", m
                 xmn = np.min(xmodl)
                 xmx = np.max(xmodl)
             else:
-                xmn = min(np.min(xtmp), np.min(xmodl))
-                xmx = max(np.max(xtmp), np.max(xmodl))
+                xmn = np.min(xtmp)
+                xmx = np.max(xtmp)
+                #xmn = min(np.min(xtmp), np.min(xmodl))
+                #xmx = max(np.max(xtmp), np.max(xmodl))
```

Comment:
Actually, this is good as is.  The x-values should be tied to the data.
It is the y-values that should take on the range of model/data which
is what I think we have now.